src/core/jsdoc-tsd-parser.ts

```
@@ -266,6 +293,7 @@ export class JSDocTsdParser {
 		this.resultItems[jsdocItem.longname].push(domInterface);
 	}
 
+	// tslint:disable-next-line:member-ordering

```

can you fix the member ordering instead of ignore this error?